clang/lib/Sema/SPIRVBuiltins.td

```diff
@@ -803,13 +807,14 @@ foreach InType = TLUnsignedInts.List in {
   }
 }
 
+
```

Comment:
Is this additional empty line is necessary?

Reply:
No

Reply:
I will remove it

Reply:
No, I reverted it